src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -14,10 +18,7 @@
 
 import org.folio.querytool.domain.dto.EntityType;
 
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
prefer non-wildcard imports

```suggestion
import java.util.List;
import java.util.Optional;
import java.util.Set;
import java.util.UUID;
```